src/main/cli/process.ts

```diff
@@ -135,10 +156,23 @@ export function cli(mainFct: () => void) {
                             dialog.showErrorBox(errorTitle, e.toString());
                             process.stderr.write(e.toString() + EOL);
                         }
+                    }).catch(() => {
+                        // ignore
                     });
                 }
-            },
-        )
-        .help()
-        .parse((_PACKAGING === "0") ? process.argv.slice(2) : process.argv.slice(1));
+            } else {
+                app.exit(0);
+            }
+        },
+    )
+    .help();
+
+/**
+ * main entry of thorium
+ * @param main main function to exec
+ * @param processArgv process.argv
+ */
+export function cli(_main: () => void, _processArgv = process.argv) {
```

Comment:
Normally, underscore-prefixed function parameters are for explicitly declaring unused local variables, so that the TypeScript linter ignores them.

Reply:
done https://github.com/readium/readium-desktop/pull/606/commits/9beff77e9daf58afef8dc2cf8e953e1e1b627b92